### PR TITLE
Update ADR to support Debian 12 Bookworm

### PR DESCRIPTION
**Pull request by ikifar2012**
For now I have removed Debian Bullseye and added Bookworm as it is already implied the Bullseye is still supported for 4 months, Although I am not sure if I should add both 
Link to Supervisor PR: https://github.com/home-assistant/supervisor/pull/4377
Link to Supervised Installer PR: https://github.com/home-assistant/supervised-installer/pull/297

### CODE REVIEW

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged/released.

**Comment by JesusFreke** [comment]
debian 12 doesn't seem to be supported yet, in as much as the latest release of the supervised-installer doesn't yet support 12. 

Lucky me, I just happen to be trying to install this in the middle of the migration xD

**Comment by ikifar2012** [comment]
@JesusFreke I am currently waiting on the new supervisor to come out before I release the updated package, till then see: https://github.com/home-assistant/supervised-installer/issues/299 for the workaround

**Pull request merged**
